source/MapDetailPanel.h

```diff
@@ -70,6 +74,9 @@ class MapDetailPanel : public MapPanel {
 	int governmentY = 0;
 	int tradeY = 0;
 
+	// Wether or not the planet view is hovered, or the trade panel (and should be affected by up and down keys).
```

Comment:
```suggestion
	// Whether or not the planet view is hovered, or the trade panel (and should be affected by up and down keys).
```
By the way, what does the second half of the sentence mean?

Reply:
Perhaps it'd be better reworded as such.
```suggestion
	// Which panel is being hovered over and should be affected by up and down keys.
```

Reply:
Yeah I agree that's clearer, so that's what I went with.